app/services/conferences/creation_service.rb

```diff
@@ -80,6 +80,7 @@ def pr_body
       Here is a new conference:
       [#{@params[:url]}](#{@params[:url]})
       #{@params[:cfpUrl].present? ? "CFP: [#{@params[:cfpUrl]}](#{@params[:cfpUrl]})" : ''}
+      #{@params[:twitter].present? ? "Twitter: [https://twitter.com/#{@params[:twitter]}](https://twitter.com/#{@params[:twitter]})" : ''}
```

Comment:
Not sure why you want to do that, the link is already clickable on confs.tech. 🤔

Reply:
Oh ok, I didn't see where you were, sorry!